packages/glibmm_2_4.rb

```diff
@@ -0,0 +1,44 @@
+require 'package'
+
+class Glibmm_2_4 < Package
```

Comment:
Wouldn't a better name be `Glibmm_2_66`?

Reply:
Same reason as with cairomm-1.0.